pkg/storage/client_replica_test.go

```diff
@@ -1208,9 +1208,8 @@ func TestRangeInfo(t *testing.T) {
 
 // TestCampaignOnLazyRaftGroupInitialization verifies expected
```

Comment:
FWIW it is no longer necessary to rename the test - removing the Skip will cause stress builder to pick it up.